spec/rubocop/cop/ezcater/rspec_require_gql_error_helpers_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "spec_helper"
+
+RSpec.describe RuboCop::Cop::Ezcater::RspecRequireGqlErrorHelpers, :config do
+  subject(:cop) { described_class.new }
+
+  let(:error_message) { described_class::MSG }
+  let(:expected_source) { "GraphQL::ExecutionError" }
+
+  it "registers an offense when attempting to directly use GraphQL::ExecutionError with options" do
+    source = "GraphQL::ExecutionError.new(\"An error occurred\", options: { status_code: 401 })"
+    inspect_source(cop, source)
+    expect(cop.messages).to eq([error_message])
+    expect(cop.highlights).to eq([expected_source])
+  end
+
+  it "registers an offense when attempting to directly use GraphQL::ExecutionError" do
+    source = "GraphQL::ExecutionError.new(\"An error occurred\")"
+    inspect_source(cop, source)
+    expect(cop.messages).to eq([error_message])
+    expect(cop.highlights).to eq([expected_source])
+  end
+
+  it "accepts usage of the GQLErrors.request_error" do
+    inspect_source(cop, "GQLErrors.request_error(\"You can't access this\", 401)")
+    expect(cop.offenses).to be_empty
+  end
+
+  it "accepts usage of the GQLErrors.field_error" do
+    inspect_source(cop, "GQLErrors.field_error(\"is invalid\", :first_name, \"First Name\")")
+    expect(cop.offenses).to be_empty
+  end
+
+  it "accepts usage of the GQLErrors.field_errors_for" do
+    inspect_source(cop, "GQLErrors.field_errors_for(my_model, context)")
+    expect(cop.offenses).to be_empty
+  end
```

Comment:
Since the cop is just concerned with `GraphQL::ExecutionError` do each of the `GQLErrors` examples add something?

Reply:
Glad you commented on this. The short answer, I'm not sure it does. I debated whether or not I should add these, and landed on it being appropriate to validate that the "good" examples aren't impacted by this rule. That said, I can appreciate that what is being tested has nothing to do with the code that is written. I'll remove for now.